Formula/ghostscript.rb

```diff
@@ -30,43 +29,55 @@ class Ghostscript < Formula
   end
 
   depends_on "pkg-config" => :build
+  depends_on "fontconfig"
+  depends_on "freetype"
+  depends_on "jbig2dec"
+  depends_on "jpeg"
+  depends_on "libidn"
```

Comment:
Why does this need `libidn`?
```
❯ brew info libidn
libidn: stable 1.37 (bottled)
International domain name library
```
Seems a bit odd 😄 ...unless it needs this to format URLs or something?

Reply:
From configure:
```
--without-libidn        Do not use libidn to support Unicode passwords
```

Reply:
Is there an alternative way to support Unicode passwords? As long as it's not ICU.

Reply:
I am not familiar enough with `libidn`.

I see some Linux distros switched to `libidn2`
e.g. Fedora: https://src.fedoraproject.org/rpms/ghostscript/c/d893ea4979088baf210c1e8f2f732aa2f81c57cd?branch=master

Reply:
I tried just changing to `libidn2` similar to Fedora/RedHat, but `configure` didn't pick it up:
```
checking for libidn with pkg-config... no
checking for stringprep in -lidn... no
```

Will need to see if possible to force linkage.

Based on comments that `libidn` should be deprecated and replaced with `libidn2`, it is worth looking into.

Reply:
We have a bunch of formulae still that use `libidn`, so we can save any migration needed for later, I think.